angular/official/list/src/app/message/message.component.html

```diff
@@ -5,7 +5,7 @@ <h2>
       {{ message.fromName }}
       <span class="date">
         <ion-note>{{ message.date }}</ion-note>
-        <ion-icon name="chevron-forward" size="small" *ngIf="isIos()"></ion-icon>
+        <ion-icon [attr.aria-hidden]="true" name="chevron-forward" size="small" *ngIf="isIos()"></ion-icon>
```

Comment:
Is there a reason why we need to do this syntax? You should be able to do `aria-hidden="true"`.

Reply:
Also `aria-hidden` is an attribute not a property, so it accepts the string `"true"` not the boolean.